lib/google_drive.rb

```diff
@@ -86,70 +84,50 @@ def self.login_with_oauth(client_or_access_token, proxy = nil)
     # +path+ defaults to ENV["HOME"] + "/.ruby_google_drive.token".
     #
     # You can specify your own OAuth +client_id+ and +client_secret+. Otherwise the default one is used.
-    def self.saved_session(path = nil, proxy = nil, client_id = nil, client_secret = nil)
+    def self.saved_session(path = './.config.json', proxy = nil, client_id = nil, client_secret = nil)
```

Comment:
Can you keep default to ENV["HOME"] + "/.ruby_google_drive.token" for backward compatibility?